common/djangoapps/enrollment/views.py

```diff
@@ -244,14 +244,16 @@ def get(self, request):
                     ).format(username=request.user.username)
                 }
             )
-        return Response([
-            {
-                "org": role.org,
-                "course_id": text_type(role.course_id),
-                "role": role.role
-            }
-            for role in roles_data
-        ])
+        return Response({
```

Comment:
This is a signature change. I know that this endpoint is not yet used, so changing it like this is fine. But please be careful of API signature changes because it breaks production if there are API consumers

Reply:
Absolutely. This is a special case where I know for certain no one is using this, but yeah, otherwise I wouldn't do it like this